File: ClientApp/src/models/Rating.js
import { Player } from "./Player";

export class Rating {
  constructor({
    id = undefined,
    player = new Player(),
    playerId = undefined,
    gameId = undefined,
    score = 2400
  } = {}) {
    this.id = id;
    this.player = player;
    this.playerId = playerId;
    this.gameId = gameId;
    this.score = score;
  }

  toJSON = () => {
    return JSON.stringify({
      id: this.id,
      playerId: this.playerId,
      gameId: this.gameId,
      score: this.score
    });
  };
}

File: Models/Rating.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace sclask.Models
{
  public class Rating
  {
    public int Id { get; set; }
    public Player Player { get; set; }
    public int PlayerId { get; set; }
    public Game Game { get; set; }
    public int GameId { get; set; }
    public float Score { get; set; }
    
    public DateTime LastUpdateDate { get; set; }
  }
}